core/capabilities/targets/write_target.go

```diff
@@ -181,12 +182,18 @@ func evaluate(rawRequest capabilities.CapabilityRequest) (r Request, err error)
 		return r, fmt.Errorf("WorkflowExecutionID in the report does not match WorkflowExecutionID in the request metadata. Report WorkflowExecutionID: %+v, request WorkflowExecutionID: %+v", reportMetadata.WorkflowExecutionID, rawRequest.Metadata.WorkflowExecutionID)
 	}
 
-	if hex.EncodeToString(reportMetadata.WorkflowOwner[:]) != rawRequest.Metadata.WorkflowOwner {
+	if !strings.EqualFold(hex.EncodeToString(reportMetadata.WorkflowOwner[:]), rawRequest.Metadata.WorkflowOwner) {
```

Comment:
Can we add comments in both cases to explain what we're doing and why we need it?

For item 1: do a case sensitive verification of the owner address (so that a check-summed address matches its non-checksummed version).

For item 2: workflowNames are padded in the core node to 10bytes